fdbserver/ClusterController.actor.cpp

```diff
@@ -2811,8 +2811,9 @@ ACTOR Future<Void> metaclusterMetricsUpdater(ClusterControllerData* self) {
 					wait(store(self->db.metaclusterMetrics,
 					           metacluster::MetaclusterMetrics::getMetaclusterMetrics(self->cx)));
 				} catch (Error& e) {
-					// Ignore errors about the cluster changing type
-					if (e.code() != error_code_invalid_metacluster_operation) {
+					// Ignore errors about the cluster changing type or unsupported metacluster versions
+					if (e.code() != error_code_invalid_metacluster_operation &&
+					    e.code() != error_code_unsupported_metacluster_version) {
```

Comment:
The cluster controller is written to function even with an incompatible metacluster registration entry. This particular code exists mainly to populate some metrics and is not critical, so we shouldn't prevent the cluster from recovering if we hit it.

Reply:
I guess probably the best thing we could do is to note that we failed to fetch the metrics and potentially report something accordingly where we would have otherwise reported the metrics